content/webapp/components/EventCard/EventCard.tsx

```diff
@@ -1,25 +1,38 @@
+import { FC } from 'react';
 import { EventBasic } from '../../types/events';
 import CompactCard from '../CompactCard/CompactCard';
 import Image from '@weco/common/views/components/Image/Image';
 import StatusIndicator from '@weco/common/views/components/StatusIndicator/StatusIndicator';
 import EventDateRange from '../EventDateRange/EventDateRange';
 import { classNames, font } from '@weco/common/utils/classnames';
 import { getCrop } from '@weco/common/model/image';
+import { dateFromDateOrString } from '@weco/common/utils/dates';
 
 type Props = {
   event: EventBasic;
   xOfY: { x: number; y: number };
 };
 
-const EventCard = ({ event, xOfY }: Props) => {
-  const DateRangeComponent = <EventDateRange event={event} />;
+const EventCard: FC<Props> = ({ event, xOfY }) => {
+  const eventTimes = event.times.map(time => ({
+    ...time,
+    range: {
+      startDateTime: dateFromDateOrString(time.range.startDateTime),
+      endDateTime: dateFromDateOrString(time.range.endDateTime),
+    },
+  }));
+  const eventWithDates = {
+    ...event,
+    times: eventTimes,
+  };
```

Comment:
Could you use `fixEventDatesInJson` instead? See https://github.com/wellcomecollection/wellcomecollection.org/blob/main/content/webapp/services/prismic/transformers/events.ts#L332-L334

It's not ideal, but it's what we use for other cases where Events have been serialised as JSON and we've lost the Date type from strings.